### PR TITLE
Implement missing features on supporters list

### PR DESCRIPTION
**Pull request by timEulitz**
T182698

### CODE REVIEW

File: skins/cat17/src/scripts/supporters.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+(function( $ ) {
+	$( document ).ready( function() {
+		initSupportersTable();
+		attachClickEvent();
+	} );
+
+	function initSupportersTable() {
+		$( '.donors tr' ).each( function() {
+			var commentColumn = $( this ).find( ':nth-child(3)' )[0];
```
[the remark this answers is not in the document]
Comment (timEulitz) [reply skins/cat17/src/scripts/supporters.js]: Yeah, that's much better actually. I have placed it in the `if` directly now to get rid of the variable altogether, not sure if that makes it harder to read / messier, though.

[the remark this answers is not in the document]
Comment (tzhelyazkova) [reply skins/cat17/src/scripts/supporters.js]: Hmm, ideally I'd extract the `if` statement in a "helper" function named something like `commentExists` which will make things more readable.  
``if ( commentExists().bind( this ) ) {
  $( this ).addClass( 'commented' );
}``

[the remark this answers is not in the document]
Comment (timEulitz) [reply skins/cat17/src/scripts/supporters.js]: @tzhelyazkova Test

File: skins/cat17/src/scripts/supporters.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+				$( this ).addClass( 'commented' );
+			}
+		} );
+	}
+
+	function attachEvents() {
+		var commentedRows = $( '.commented' );
+		commentedRows.click( function() {
+			if( $( this ).hasClass( 'active' ) ) {
+				hideComment.call( $( this ) )
```
Comment (gbirke) [inline skins/cat17/src/scripts/supporters.js]: By using `.call`, you set the `this` value inside the `showComment` and `hideComment` functions to `$(this)` (from `attachEvents`). Inside the functions you do a `$(this)` which is a no-op, as jQuery will just return the object. Why not use `$( this )` as a `tableRow` parameter?

File: skins/cat17/src/scripts/supporters.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+		$( '.donors tbody tr' ).each( function() {
+			if( getComment( this ) ) {
+				$( this ).addClass( 'commented' );
+			}
+		} );
+	}
+
+	function attachEvents() {
+		var commentedRows = $( '.commented' );
+		commentedRows.click( function() {
```
Comment (JeroenDeDauw) [inline skins/cat17/src/scripts/supporters.js]: /me is TRIGGERED by avoidable variable

Comment (timEulitz) [reply skins/cat17/src/scripts/supporters.js]: Nope, this was an oversight, thanks for pointing it out.

**Pull request merged**